src/timelines/preamble.js

```diff
@@ -1,25 +1,21 @@
-import htmlKeyboardResponse from "@jspsych/plugin-html-keyboard-response";
-
 import { config } from "../config/main";
 
-import { enterFullscreen } from "../trials/fullscreen";
-import holdUpMarker from "../trials/holdUpMarker";
-import startCode from "../trials/startCode";
-import { showName, showWelcome } from "../trials/welcome";
+import { enterFullscreenTrial } from "../trials/fullscreen";
+import { holdUpMarkerTrial } from "../trials/holdUpMarker";
+import { startCodeTrial } from "../trials/startCode";
+import { welcomeTrial, nameTrial } from "../trials/welcome";
+
+/** Builds the blocks of trials needed to start and setup the experiment */
+function buildPreambleBlock() {
+  const timeline = [nameTrial, enterFullscreenTrial, welcomeTrial];
```

Comment:
I like this pattern a lot! You're basically moving things from the top level into functions, right?

Reply:
Right. And there's a HUGE difference between code that's inside a function and code that's inside a trial's callback function. Hoping to be a little more clear about that.